cantools/database/can/message.py

```diff
@@ -2,12 +2,14 @@
 
 import binascii
 from copy import deepcopy
+import textwrap
```

Comment:
this could be a problematic dependency, see #280. in particular because this file this affects everything instead of just a single subparser.

Reply:
[Textwrap](https://docs.python.org/3/library/textwrap.html) is part of the standard library, so it shouldn't cause any problems.

Reply:
right. (please forget what I said ;))